components/bms_worker/include/BatteryMonitoring.h

```diff
@@ -65,6 +65,9 @@ typedef struct
     float       chargeLimit;
     float       dischargeLimit;
 
+    float       heatDischargeCurrentLimit;
+    float       heatChargeCurrentLimit;
+
```

Comment:
The structure of this is not fully thought out, you now have 2 current limits for each of charge and discharge